packages/react/src/components/Button/Button.stories.mdx

```diff
@@ -194,6 +208,44 @@ On through of which no the your phase more know anyone tried he when line me wha
     </Story>
 </Canvas>
 
+### Knapp som laster
+<Canvas>
```

Comment:
Suggest adding a line or two about using `aria-disabled` instead of `disabled` for loading buttons, but don't worry if you can't come up with the wording, We can update it later if needed.